pkg/kubectl/rollout_status.go

```diff
@@ -131,13 +131,13 @@ func (s *StatefulSetStatusViewer) Status(namespace, name string, revision int64)
 	if sts.Status.ObservedGeneration == nil || sts.Generation > *sts.Status.ObservedGeneration {
 		return "Waiting for statefulset spec update to be observed...\n", false, nil
 	}
-	if sts.Status.ReadyReplicas < sts.Spec.Replicas {
-		return fmt.Sprintf("Waiting for %d pods to be ready...\n", sts.Spec.Replicas-sts.Status.ReadyReplicas), false, nil
+	if sts.Status.ReadyReplicas < *sts.Spec.Replicas {
```

Comment:
Can `sts.Spec.Replicas` be nil? Then we should check that before to avoid a panic.

Reply:
Sam below with the Partition and again Replicas.

Reply:
@sttts Thanks, I have made the update.